### ISSUE TITLE
Has Gab Not Been Updated for 9 months?

### ISSUE THREAD

**Issue by teihome** [GabOpenSource/gab-social#5]
According to AGPL, the very version that is served, needs to be open source.

Every Update to the Gab Website, Needs an corresponding update to the source code here.

**Comment by btakita** [comment]
> The GNU Affero General Public License is designed specifically to ensure that, in such cases, the modified source code becomes available to the community. It requires the operator of a network server to provide the source code of the modified version running there to the users of that server. Therefore, public use of a modified version, on a publicly accessible server, gives the public access to the source code of the modified version.

Yes. It would be great to contribute to the project. The UX does need some love.

**Comment by tux2bsd** [comment]
> Gab Social is open source software. code.gab

Disingenuous use of "is", "was" would be the correct word.

I didn't even bother downloading the old zip file.

Interesting the code isn't pushed to github using git.

**Comment by reiphoNi9iey3qu** [comment]
> Gab Social is open source software. code.gab

~~Disingenuous use of "is", "was" would be the correct word.~~

I didn't even bother downloading the old zip file.

Interesting the code isn't pushed, as normal code, to github using git.

**Comment by teihome** [comment]
@tux2bsd if they do not publish the version that they serve, they are breaking the terms of the license and are liable.

**Comment by tux2bsd** [comment]
@telhome that would mean that "is" is the right word AND they must post code, like you said originally.

**Comment by kjkrum** [comment]
@tux2bsd I tried uploading the un-archived code and GitHub blocked it for containing user secrets. I reported it to the author and they ignored me. I think it's a false detection; the secrets appear to be temporary/disposable credentials used to access something in the build environment that probably doesn't need to be restricted in the first place. Still, the code is not buildable as is.

**Comment by nilFinx** [comment]
From [here](https://github.com/GabOpenSource/gab-social/issues/3#issuecomment-2080382678), this repository does not accept any contributions. The OP's question is still valid though.
